scripts/commit-msg.js

```diff
@@ -13,7 +13,7 @@ const commitEditMsgFile = '.git/COMMIT_EDITMSG'
 if (fs.existsSync(commitEditMsgFile)) {
   try {
     const commitMessage = fs.readFileSync(commitEditMsgFile, 'utf8')
-    const regex = '(build|BREAKING|chore|ci|docs|feat|fix|perf|refactor|style|test)(\\((web app|server|hotword|skill\\/([\\w-]+)))?\\)?: .{1,50}' // eslint-disable-line no-useless-escape
+    const regex = '(build|BREAKING|chore|ci|docs|feat|fix|perf|refactor|style|test)(\\((web app|docker|server|hotword|skill\\/([\\w-]+)))?\\)?: .{1,50}' // eslint-disable-line no-useless-escape
```

Comment:
Not strictly related to the fix, but I think a "docker" scope for commits messages is nice to have, if you don't want to have this scope, I can revert this change.